mono/metadata/marshal.c

```diff
@@ -149,6 +149,17 @@ init_safe_handle ()
 		mono_defaults.safehandle_class, "DangerousRelease", 0);
 }
 
+G_GNUC_UNUSED static void
+code_for (MonoMethod *method) {
```

Comment:
This should go into the debug helpers source, prefixed with mono_ and documented in our website.